stm32_loader.py

```diff
@@ -245,6 +254,7 @@ def load(port,
          progress_report_callback=None,
          readout_unprotect=False,
          write_unprotect=False,
+         go=True,
```

Comment:
I would suggest to keep it disabled by default, in order to stay consistent with other optional features like `readout_unprotect` and `write_unprotect`.